index.js

```diff
@@ -1,4 +1,3 @@
-// tslint:disable:no-console
```

Comment:
Updated balena-lint inherits the ts rules for js which means we don't need to manually disable this for js anymore

Reply:
Seems I'm getting errors locally without this:

```
index.js:10:3
ERROR: 10:3  no-console  Calls to 'console.log' are not allowed.
ERROR: 12:4  no-console  Calls to 'console.log' are not allowed.
ERROR: 17:4  no-console  Calls to 'console.log' are not allowed.
```